core/src/mindustry/world/blocks/payloads/PayloadMassDriver.java

```diff
@@ -122,8 +122,8 @@ public void drawPlace(int x, int y, int rotation, boolean valid){
     }
 
     @Override
-    public TextureRegion[] makeIconRegions(){
-        return new TextureRegion[]{leftRegion, rightRegion, capRegion};
+    public void getRegionsToOutline(Seq<TextureRegion> out){
```

Comment:
Why was this changed - was it tested to make sure things don't break?

Reply:
I legitimately do not remember

Reply:
![969375577112993912](https://github.com/Anuken/Mindustry/assets/10100000/6130eaa1-4e7f-47d5-be7a-4f75c32d84ce)

Reply:
can you revert it

Reply:
> Why was this changed - was it tested to make sure things don't break?

Oh, now I remember, it's literally in the commit name.
Since `makeIconRegions` is vanilla-only, modded ones wouldn't get outlined.
...I'm gonna go test real quick.

Reply:
Yep. Reverted my reversion.

Before change:
<img width="331" alt="20240624-214407-javaw" src="https://github.com/Anuken/Mindustry/assets/54301439/6fcdb7c1-dcc2-43aa-87da-8a7827551c98">

After change:
<img width="278" alt="20240624-214506-javaw" src="https://github.com/Anuken/Mindustry/assets/54301439/c09d5aea-a63f-4380-b660-93cb4a7a922a">